binding/python/setup.py

```diff
@@ -35,6 +46,7 @@
 include pvrhino/lib/mac/arm64/libpv_rhino.dylib
 recursive-include pvrhino/lib/raspberry-pi *
 include pvrhino/lib/windows/amd64/libpv_rhino.dll
+include pvrhino/lib/windows/amd64/pv_ypu_impl_cuda.dll
```

Comment:
not a big deal, but we could also do a `recursive-include` to future proof it a bit